src/test/java/com/wikia/webdriver/testcases/mercurytests/SearchTests.java

```diff
@@ -180,4 +185,55 @@ public void mercury_search_redirectToNewResultsPageFromNoResults() {
     Assertion.assertFalse(searchResults.isNoResultsPagePresent());
     Assertion.assertTrue(searchResults.areResultsPresent());
   }
+
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
+  @Test(groups = "mercury_search_defaultResultsNumberOnSearchResultsPage")
+  public void mercury_search_defaultResultsNumberOnSearchResultsPage() {
+    SearchResultsPage resultsPage =
+        new SearchResultsPage()
+            .openForQuery(MULTIPLE_RESULTS_SEARCH_PHRASE);
+
+    Assertion.assertEquals(resultsPage.getResultCardsNumber(), SEARCH_RESULTS_DEFAULT_NUMBER);
+  }
+
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
+  @Test(groups = "mercury_search_loadingMoreResultsOnSearchResultsPage")
+  public void mercury_search_loadingMoreResultsOnSearchResultsPage() {
+    SearchResultsPage resultsPage =
+        new SearchResultsPage()
+            .openForQuery(MULTIPLE_RESULTS_SEARCH_PHRASE);
+
+    int defaultCardNumber = resultsPage.getResultCardsNumber();
+    
+    Assertion.assertTrue(resultsPage.isLoadMoreButtonVisible());
+    Assertion.assertEquals(defaultCardNumber, SEARCH_RESULTS_DEFAULT_NUMBER);
+
+    resultsPage.clickLoadMoreButton();
+    int moreResultsLoaded = resultsPage.getResultCardsNumber() - defaultCardNumber;
+
+    Assertion.assertEquals(moreResultsLoaded, SEARCH_RESULTS_DEFAULT_NUMBER);
+    Assertion.assertTrue(resultsPage.isLoadMoreButtonVisible());
+  }
+
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
+  @Test(groups = "mercury_search_loadMoreResultsOnSearchResultsPageNotVisible")
+  public void mercury_search_loadMoreResultsOnSearchResultsPageNotVisible() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)